src/wasm32_bindgen.rs

```diff
@@ -102,12 +102,20 @@ extern "C" {
     #[wasm_bindgen(method, js_name = getRandomValues, structural)]
     fn get_random_values(me: &BrowserCrypto, buf: &mut [u8]);
 
-    #[wasm_bindgen(js_name = require)]
-    fn node_require(s: &str) -> NodeCrypto;
-
     #[derive(Clone, Debug)]
     type NodeCrypto;
 
     #[wasm_bindgen(method, js_name = randomFillSync, structural)]
     fn random_fill_sync(me: &NodeCrypto, buf: &mut [u8]);
 }
+
```

Comment:
Formatting nit: could we just put all of the Node.js declarations in their own block. Something like:

```rust
#[wasm_bindgen]
extern "C" {
    type NodeModule;
    #[wasm_bindgen(js_name = module)]
    static MODULE: NodeModule;

    #[derive(Clone, Debug)]
    type NodeCrypto;
    #[wasm_bindgen(method)]
    fn require(this: &NodeModule, s: &str) -> NodeCrypto;

    #[wasm_bindgen(method, js_name = randomFillSync)]
    fn random_fill_sync(this: &NodeCrypto, buf: &mut [u8]);
}
```